### PR TITLE
Task2c, created index.php page and filed with dummy data ready for viewHelper

### PR DESCRIPTION
**Pull request by turnerror**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+</head>
+<body>
+<nav class="navbar navbar-light bg-light mb-4">
+    <div class="container">
+        <a class="navbar-brand" href="#">
+            <img src="./img/logo.png" height="40"
+                 class="d-inline-block align-top" alt="">
+        </a>
+        <h2>Armadillo Estates</h2>
+        <h2></h2>
```
Comment (amandahrdmn) [inline index.php]: What's this for?

File: src/Hydrators/PropertyHydrator.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    private $database;
+
+   public function __construct(\PDO $database)
+   {
+       $this->database = $database;
+   }
+
+   public function getAllBasicProperties()
+   {
+       $query = $this->database->query("SELECT `Address1`, `Town`, `Type`, `Status`, `id`,  `Image` FROM `properties`;");
```
Comment (amandahrdmn) [inline src/Hydrators/PropertyHydrator.php]: same comments as were in task2b pr

**Review by amandahrdmn — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (turnerror) [reply index.php]: The parent element has justify-content spacebetween on it. And its the only easy way for having the Logo on the left and then title in the middle at the top of the page

File: src/Hydrators/PropertyHydrator.php (hunk 2 above)
Comment (turnerror) [reply src/Hydrators/PropertyHydrator.php]: yeah this won't be in this PR once Task2b is in Story1

**Review by amandahrdmn — APPROVED** [review]

**Pull request merged**